lib/codebreaker/classes/player.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Codebreaker
+  # player entity
+  class Player
```

Comment:
Why do we need class Player?
I can't figure out any reasons to make it

Reply:
I would be complicated without Player. I think game MUST has something like user or player and difficulty, and like additional rules field(but I don't like idea. with rules). Game class need Player class, it's more OOP.  Thanks for your question.